core/block/service.go

```diff
@@ -390,8 +390,28 @@ func (s *Service) prepareDetailsForInstallingObject(ctx context.Context, spaceID
 			relations[i] = id
 		}
 		newDetails.Fields[bundle.RelationKeyRecommendedRelations.String()] = pbtypes.StringList(relations)
+	}
+
+	objectTypes := pbtypes.GetStringList(newDetails, bundle.RelationKeyRelationFormatObjectTypes.String())
 
+	if len(objectTypes) > 0 {
+		for i, ot := range objectTypes {
```

Comment:
please use full names for variables
https://linear.app/anytype/issue/GO-1920/faulty-syncing-and-remote-storage#comment-4399fb8d

Reply:
Yeah, thank you for reminder!
Fixed